webpack/assets/javascripts/react_app/routes/Models/ModelsPage/ModelsPage.js

```diff
@@ -20,47 +17,30 @@ const ModelsPage = ({
   hasError,
   itemCount,
   message,
-  canCreate,
-}) => {
-  const handleSearch = query => fetchAndPush({ searchQuery: query, page: 1 });
-
-  const createBtn = (
-    <Link to="/models/new">
-      <Button>{__('Create model')}</Button>
-    </Link>
-  );
-
-  return (
-    <PageLayout
-      header={__('Hardware models')}
-      searchable={!isLoading}
-      searchProps={MODELS_SEARCH_PROPS}
-      searchQuery={search}
-      isLoading={isLoading && hasData}
-      onSearch={handleSearch}
-      onBookmarkClick={handleSearch}
-      toolbarButtons={canCreate && createBtn}
-    >
-      <ModelsPageContent
-        models={models}
-        page={page}
-        perPage={perPage}
-        search={search}
-        sort={sort}
-        hasData={hasData}
-        hasError={hasError}
-        isLoading={isLoading}
-        itemCount={itemCount}
-        fetchAndPush={fetchAndPush}
-        message={message}
-      />
-    </PageLayout>
-  );
-};
+}) => (
+  <PageLayout
+    apiUrl={MODELS_API_PATH}
+    apiOptions={{ key: API_REQUEST_KEY }}
+    header={__('Hardware models')}
+    controller="models"
+  >
+    <ModelsPageContent
+      models={models}
+      page={page}
+      perPage={perPage}
```

Comment:
Is it OK to set this PR to be blocked on #8990 which removed some code from the models page?

Reply:
np